frontend/packages/dev-console/src/components/helm/list/HelmReleaseList.tsx

```diff
@@ -13,22 +15,83 @@ import HelmReleaseListRow from './HelmReleaseListRow';
 import HelmReleaseListHeader from './HelmReleaseListHeader';
 import { HelmRelease } from '../helm-types';
 
+import './HelmReleaseList.scss';
+
 interface HelmReleaseListProps {
   namespace: string;
   secrets?: FirehoseResult;
 }
 
 const HelmReleaseList: React.FC<HelmReleaseListProps> = ({ namespace, secrets }) => {
-  const memoizedSecrets = useDeepCompareMemoize(secrets?.data);
+  const secretsCountRef = React.useRef<number>(0);
+  const newCount = secrets.data?.length ?? 0;
```

Comment:
Sometimes on page reload, secrets is undefined which throws error and breaks the UI. Probably better to use optional chaining wherever properties of `secrets` are being accessed.

![Screenshot from 2020-05-13 00-20-07](https://user-images.githubusercontent.com/6041994/81734044-8f722980-94b0-11ea-8818-58d7f7319850.png)

Reply:
@jeff-phillips-18 ^